.travis.yml

```diff
@@ -12,6 +12,8 @@ before_install:
 
 install:
   - yarn install --no-lockfile
+  - npm install openlayers@^3 --save-dev
```

Comment:
we're using `yarn` in this project, please avoid `npm` here. 

also, why `--save-dev` in the CI config instead of adding it as a dev dep in `package.json`?

Reply:
You're right, the `--save-dev` isn't necessary here; I'll remove it.

The use of `yarn` here had confused me, since it isn't in the development dependencies.  Is there a reason for that?  In this project it looks like it's only used in Travis (I'd even thought about proposing to replace it with just `npm`!).  Should its use perhaps be documented in the README?